system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -364,17 +386,28 @@ export class TopSiteList extends React.PureComponent {
     const topSitesUI = [];
     for (let i = 0, l = props.TopSitesCount; i < l; i++) {
       const link = topSites[i];
-      topSitesUI.push(!link ? <TopSitePlaceholder key={i} index={i} onDragEvent={this.onDragEvent} /> : <TopSite
-        key={i}
-        dispatch={props.dispatch}
-        link={link}
-        index={i}
-        intl={props.intl}
-        onEdit={props.onEdit}
-        onDragEvent={this.onDragEvent} />);
+      topSitesUI.push(!link ? (
+        <TopSitePlaceholder
+          key={i}
+          index={i}
+          onDragEvent={this.onDragEvent}
+          dispatch={props.dispatch}
+          intl={props.intl} />
```

Comment:
All of these props are shared by `<TopSite>` below, so you can use a shared `<TopSitePlaceholder {...siteProps} />`